multiqc_bcbio/bcbio.py

```diff
@@ -324,23 +331,23 @@ def _bcbio_umi_table(self, parsed_data):
                                         'description': 'Count of UMI consensus reads mapped',
                                         'format': '{:n}'}
         keys['umi_consensus_pct'] = {'title': 'Consensus reduction',
-                                        'description': 'Percent of original reads removed by consensus',
-                                        'format': '{:.1f}%'}
+                                     'description': 'Percent of original reads removed by consensus',
+                                     'format': '{:,.1f}%'}
```

Comment:
Same as above with `suffix` and removing `%` from the format string. Apologies, need to improve the docs on this I think.